components/tyk-dashboard/README.md

```diff
@@ -0,0 +1,91 @@
+## Tyk Dashboard
+[Tyk](https://tyk.io) is an open source Enterprise API Dashboard, supporting REST, GraphQL, TCP and gRPC protocols.
+
+Tyk Dashboard is provided ‘Batteries-included’, with no feature lockout. Enabling your organization to control who accesses your APIs, when they access, and how they access it. Tyk Dashboard can also be deployed as part of a larger Full Lifecycle API Management platform Tyk Self-Managed which also includes Management Control Plane, Dashboard GUI and Developer Portal.
```

Comment:
The [Tyk Dashboard](https://tyk.io/docs/tyk-dashboard/) is the GUI and analytics platform for Tyk. It provides an easy-to-use management interface for managing a Tyk installation as well as clear and granular analytics.

The Dashboard also provides the [API Developer Portal](https://tyk.io/docs/tyk-developer-portal/), a customisable developer portal for your API documentation, developer auto-enrolment and usage tracking.